package.json

```diff
@@ -2,13 +2,16 @@
   "name": "backpat",
   "version": "0.5.0",
   "description": "A simple tool for high-fiving your tech stack",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "nyc mocha",
+    "prebuild": "rimraf dist",
+    "build": "npm-run-all --parallel build:*",
+    "build:main": "babel --out-dir dist --ignore *.test.js src",
```

Comment:
Basically, use babel to transpire to dist directory from src directory. Ignore any files that end in test.js.